frontend/src/components/common/StyledText/NegativeText.tsx

```diff
@@ -0,0 +1,11 @@
+import { ReactNode } from 'react'
+
+interface NegativeTextProps {
+  children?: ReactNode
+}
+
+const negativeTextStyle = { color: '#64707D' }
```

Comment:
Is the intention to make use of `$grey-700`? If so, would it be better to directly reference that variable?

Reply:
I see. You saw that `grey-700` matches this colour? Thanks for the catch!

Let me change it tomorrow.

Reply:
Yup, if you search the hex code you should find that variable in `frontend/src/styles/base/_colours.scss`. 

But I think the intention is more important. Did you intend to make use of the app's colour scheme? 🤔

Reply:
My intention is to go for the easiest way out. But reusing `grey-700` works for me too. I've refactored accordingly.